components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import BlockableLink from 'components/admin_console/blockable_link';
+import AlertBanner from 'components/alert_banner';
+
+import privateCloudImage from 'images/private-cloud-image.svg';
+
+export const contactSalesCard = (contactSalesLink: any, isFreeTrial: boolean, isPaidTier: boolean) => {
+    const title = {id: '', defaultMsg: ''};
+    const description = {id: '', defaultMsg: ''};
+
+    if (!isPaidTier) {
+        title.id = 'admin.billing.subscription.privateCloudCard.title';
+        title.defaultMsg = 'Looking for a high-trust private cloud?';
+
+        description.id = 'admin.billing.subscription.privateCloudCard.description';
+        description.defaultMsg = 'If you need software with dedicated, single-tenant architecture, Mattermost Private Cloud (Beta) is the solution for high-trust collaboration.';
+    } else if (isFreeTrial) {
+        title.id = 'admin.billing.subscription.privateCloudCard.freeTrial.title';
```

Comment:
Don't think this has a matching translation in en.json, could just be cmd + f failing